pyproject.toml

```
@@ -31,7 +31,7 @@ coveralls = "coveralls.cli:main"
 python-coveralls = "coveralls.cli:main"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = ">=3.8"
```

```suggestion
python = "^3.8"
```

This might be a better fit because it would allow any Python version 3.x that is newer than 3.8, but would intentionally break at any higher major version, such as Python 4.x, even if this is not yet foreseeable.

---

Yes, good idea. Changed as suggested.

---

CI didn't like this so I had to change this back.

---

What about 
```suggestion
python = ">=3.8,<4.0"
```

---

Python 4 is unlikely:

* https://www.techrepublic.com/article/programming-languages-why-python-4-0-will-probably-never-arrive-according-to-its-creator/
* https://web.archive.org/web/20220906155615/https://twitter.com/gvanrossum/status/1306082472443084801
* https://discuss.python.org/t/python-3-13-alpha-1-contains-breaking-changes-whats-the-plan/37490/11

Even if there is, it's too late - there are already coveralls releases without the upper pin, and pip will end up installing those really old versions.

I recommend `">=3.8"`.

---

Apparently, there's another PR #464 tackling this. And at least 3 issues: #449, #523 and #549.